src/Language/PureScript/Errors.hs

```diff
@@ -474,25 +465,30 @@ prettyPrintSingleError full level e = prettyPrintErrorMessage . positionHintsFir
     renderSimpleErrorMessage (UnknownDataConstructor dc tc) =
       line $ "Unknown data constructor " ++ showQualified runProperName dc ++ foldMap ((" for type constructor " ++) . showQualified runProperName) tc
     renderSimpleErrorMessage (UnknownImportType mn name) =
-      line $ "Module " ++ runModuleName mn ++ " does not export type " ++ runProperName name
+      line $ "The type " ++ runProperName name ++ " is not exported by the module " ++ runModuleName mn
```

Comment:
How about phrasing this one like the `UnknownExportModule` error. Something along the line of

```
Cannot import the type X from module Y
It either does not exist or the module does not export the type
```

In an ideal world also followed by:

```
Did you mean:
  X'
  X''
  X'''
```

:wink: But that's for a later date I think.